network/src/commonMain/kotlin/com/wire/kalium/network/api/conversation/ConversationEvent.kt

```diff
@@ -1,65 +1,23 @@
-    @SerialName("access") val access: List<ConversationAccess>, // How users can join conversations = ['private', 'invite', 'link', 'code']
-    @SerialName("access_role") val conversationAccess: ConversationAccess,
-    @SerialName("code") val code: String, // Stable conversation identifier ,
-    @SerialName("conversation_role") val conversationRole: String?,
-    @SerialName("creator") val creator: String,
-    @SerialName("data") val `data`: String?, // Extra (symmetric) data (i.e. ciphertext, Base64 in JSON) that is common with all other recipients. ,
-    @SerialName("email") val email: String?,
-    @SerialName("hidden") val hidden: Boolean?,
-    @SerialName("hidden_ref") val hiddenRef: String?,
-    @SerialName("id") val id: String?,
-    @SerialName("key") val key: String, // Stable conversation identifier
-    @SerialName("last_event") val lastEvent: String?,
-    @SerialName("last_event_time") val lastEventTime: String?,
-    @SerialName("members") val members: ConversationMembersResponse,
-    @SerialName("message") val message: String?,
-    @SerialName("message_timer") val messageTimer: Int?, // Per-conversation message timer (can be null)
-    @SerialName("name") val name: String,
-    @SerialName("otr_archived") val otrArchived: Boolean?,
-    @SerialName("otr_archived_ref") val otrArchivedRef: String?,
-    @SerialName("otr_muted_ref") val otrMutedRef: String?,
-    @SerialName("otr_muted_status") val otrMutedStatus: Int?,
-    @SerialName("qualified_id") val qualifiedId: ConversationId,
-    @SerialName("qualified_recipient") val qualifiedRecipient: UserId,
-    @SerialName("qualified_target") val qualifiedTarget: UserId,
-    @SerialName("qualified_user_ids") val qualifiedUserIds: List<UserId>,
-    @SerialName("receipt_mode") val receiptMode: Int, //Conversation receipt mode
-    @SerialName("recipient") val recipient: String,
-    @SerialName("sender") val sender: String,
-    @SerialName("status") val status: String, //  ['started', 'stopped']
-    @SerialName("target") val target: String?,
-    @SerialName("team") val team: String?,
-    @SerialName("text") val text: String, // The ciphertext for the recipient (Base64 in JSON)
-    @SerialName("type") val type: Int, //  ['0', '1', '2', '3']
-    @SerialName("uri") val uri: String?,
-    @SerialName("users") val users: List<SimpleMember>
+data class ConversationMembers(
+    @SerialName("user_ids") val userIds: List<String>, // Role name, between 2 and 128 chars, 'wire_' prefix is reserved for roles designed by Wire (i.e., no custom roles can have the same prefix)
+    @SerialName("users") val users: List<ConversationMember>
 )
 
 @Serializable
-data class SimpleMember(
-    @SerialName("conversation_role") val conversationRole: String?, // Role name, between 2 and 128 chars, 'wire_' prefix is reserved for roles designed by Wire (i.e., no custom roles can have the same prefix)
-    @SerialName("qualified_id") val qualifiedId: UserId
+data class ConversationUsers(
```

Comment:
I think this name is a bit confusing, `ConversationUsers` makes me think this is a list of some kind `User` object but it's actually a list of userIDs referring to members in a conversation.

Would it make sense to rename it to  `ConversationMemberIDs`

Reply:
Agree, Actually in this layer naming is a little bit confusing totally, I like to discuss the whole naming in this layer with the team - we have many unclear naming in it!, but for now I decided to keep it like that, till we agree on new convention]